src/styles/globals.css

```diff
@@ -765,3 +764,28 @@ main {
 .selected-start-end {
   background-color: var(--mui-palette-primary-main) !important;
 }
+
+.input_search div:first-child {
+  height: 44px;
+  background: #ededed;
+  padding-left: 16px;
+  border-radius: 100px;
+  border: unset !important;
+}
+.input_search fieldset {
+  border: unset !important;
+}
+
+.input_search input {
+  color: #4d4639;
+  font-size: 14px;
+}
+.color-FF {
```

Comment:
@shreyas1434shinde why these colors are here, shouldn't it be in theme variables?